fastify.js

```diff
@@ -163,6 +163,7 @@ function build (options) {
 
   // schemas
   fastify.addSchema = addSchema
+  fastify.getAllSchemas = getAllSchemas
```

Comment:
can you just do `this._schemas.getAllSchemas.bind(this._schemas)` instead?

Reply:
Maybe this function could be `getSchemas()`.

Reply:
Or if we might want to be more descriptive, `getRegisteredSchemas`.